picard/script.py

```diff
@@ -425,7 +426,8 @@ def func_unset(parser, name):
     # Allow wild-card unset for certain keys
     if name in ('performer:*', 'comment:*', 'lyrics:*'):
         name = name[:-1]
-        for key in parser.context.keys():
+        key_list = list(parser.context.keys())
```

Comment:
Can't do that since we are modifying parser.context in the loop. Storing it in an initial list just to be safe.